configs/configs_test.go

```diff
@@ -16,30 +16,37 @@ package configs
 
 import (
 	"encoding/json"
+	"fmt"
 	"testing"
 	"time"
 )
 
 func TestJSONDuration(t *testing.T) {
-	var s struct {
-		Dur JSONDuration
+	testCases := []struct {
+		json         []byte
+		wantErr      string
```

Comment:
This is tying the test to error messages we don't own. I'd be ok with an `wantErr bool` and a `err != nil` check. We have a bunch of tests like that, eg, [noop_test.go](https://github.com/google/trillian/blob/8613dac42d8a73cbf504a2c64ce36001075ceb4c/quota/noop_test.go#L38).